build.gradle

```diff
@@ -62,38 +53,38 @@ sourceSets {
 	}
 }
 
-/**
-* Gradle uses this built-in function to locate declared
-* dependencies within Maven's central repositories.
-*/
+ /**
```

Comment:
This slash in the first line of a multi-line comment does not get indented with a space.

Reply:
Finished & pushed